Tests/kaas/kaas-sonobuoy-tests/kind_config.yaml

```diff
@@ -3,6 +3,18 @@ apiVersion: kind.x-k8s.io/v1alpha4
 networking:
   apiServerAddress: 127.0.0.1
   apiServerPort: 6443
+kubeadmConfigPatches:
+  - |
+    apiVersion: kubeadm.k8s.io/v1beta3
+    kind: ClusterConfiguration
+    metadata:
+      name: config
+    apiServer:
+      extraArgs:
+        enable-admission-plugins: "NodeRestriction,PodSecurity"
+        oidc-issuer-url: https://localhost:8443
+        oidc-client-id: kubernetes
+        oidc-username-claim: sub
```

Comment:
unfortunately since kind and kubeadm are different things, we cannot leave the configuration here.
Imo this needs to be addressed in the implementation node section in another issue/PR. Please exclude it here.

Reply:
I included these settings because they are necessary to ensure that certain tests pass against kind. However, if you'd prefer to address this configuration in the implementation node section in another PR/issue, I will remove it from this PR as you suggested.